### PR TITLE
Add homepage CSS

### LINKED ISSUE #81: Styling for Homepage (not including mobile responsive styling)

### PR DESCRIPTION
**Pull request by seongryoo**
Adds basic styling to homepage, including:

- Classy whitespace!
- Font size correction
- Layout & container definition
- Hero banner styling
- Basic call-to-action styling

CSS guide: where are all the changes, anyway?

- Layouts & containers -> client/src/css/layout.css
- Hero banner dimensions/DOM positions-> client/src/css/module.css
-- (Note, module.css is meant for re-usable code blocks, like navbars, buttons, etc. I've put the hero banner styling there, perhaps controversially, because we might have multiple hero banners on our website)
- Styling (font size, line height, colors, etc.) -> client/src/css/style.css
- Whitespace adjustments -> client/src/css/layout.css (see the class .row)

image

### CODE REVIEW

**Comment by 9353oasis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Files in the [`client`](/client) directory were modified; please update the client version.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f614934f72fc697176d4b6bb4a2405697aefe972
</p>

File: client/src/components/home/Home.js [hunk 1]
```diff
@@ -9,36 +9,61 @@ import Veggies from "./img/veggies_thinking.png";
...
+        <Grid columns={"2"} verticalAlign={"middle"} stackable>
+            {/*Hero banner*/}
             <Grid.Row className="hero">
-                <Grid.Column>
-                    <Header>Do I live in a food desert?</Header>
+
+                <Grid.Column className="info-left info">
+                    <Header>Do I live in an Atlanta food desert?</Header>
+                    <p className="hero-text">Scroll down to learn more about how <i className="emph"> where you live in Atlanta can affect the food you eat.</i></p>
+                    <Button className="main-cta cta">View our map</Button>
```
Comment (evan10s) [inline client/src/components/home/Home.js]: You can make this take the user to the wizard by adding an `as={Link}` (`Link` is a Semantic UI React component, so make sure you import it) and a `to` prop in React with a value of `/map`

Also better button text might be along the lines of "find out now"

**Comment by 9353oasis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Files in the [`client`](/client) directory were modified; please update the client version.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f614934f72fc697176d4b6bb4a2405697aefe972
</p>

File: client/src/components/home/Home.js [hunk 2]
```diff
@@ -9,36 +9,61 @@ import Veggies from "./img/veggies_thinking.png";
...
             </Grid.Row>
-            <Grid.Row columns={1}>
-                <Grid.Column as={Header} size={"large"}><Button>Find out now</Button></Grid.Column>
+
+
+            {/*Call to action*/}
+            <Grid.Row columns={1} className="cta-row full-width-row">
+                <Grid.Column className="info-center info">
+                    <Header size="large">Am I living in a food desert?</Header>
+                    <Button className="main-cta cta">Find out now</Button>
```
Comment (evan10s) [inline client/src/components/home/Home.js]: We can make this a link to /map now as well

File: client/src/css/module.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
+/* Navbar module */
+.ui.menu {
+    /*box-shadow: none;*/
```
Comment (evan10s) [inline client/src/css/module.css]: remove comment pls

**Review by evan10s — CHANGES_REQUESTED** [review]
Wow, this looks great!  Just a couple minor things to fix.  Also, can you add a screenshot to the PR description?

File: client/src/components/home/Home.js (hunk 1 above)
Comment (seongryoo) [reply client/src/components/home/Home.js]: Got it!

File: client/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fd-client",
-  "version": "0.7.1",
+  "version": "0.8.1",
```
Comment (evan10s) [inline client/package.json:3]:
```suggestion
  "version": "0.8.0",
```

**Review by evan10s — APPROVED** [review]
Looks good, just commit the one version number suggestion I made and then you can merge when github lets you

**Pull request merged**